RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -221,23 +247,46 @@
         maxLostLayers = [4,3,2],
         maxDz         = [0.5,0.35,0.2],
         maxDr         = [0.3,0.2,0.1]
-	),
+    ),
     vertices = 'firstStepGoodPrimaryVertices'
 )
-from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
+jetCoreRegionalStepBarrel = jetCoreRegionalStep.clone(
+    src = 'jetCoreRegionalStepBarrelTracks',
+    mva = dict(
+#	minPixelHits  = [1,1,1], # they could be easily increased to at least 2 or 3 !
+        min3DLayers   = [1,2,2],
+    ),
+)
 
+from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
 trackingPhase1.toReplaceWith(jetCoreRegionalStep, TrackMVAClassifierPrompt.clone(
      mva = dict(GBRForestLabel = 'MVASelectorJetCoreRegionalStep_Phase1'),
      src = 'jetCoreRegionalStepTracks',
      qualityCuts = [-0.2,0.0,0.4]
 ))
 
+trackingPhase1.toReplaceWith(jetCoreRegionalStepBarrel, jetCoreRegionalStep.clone(
+     src = 'jetCoreRegionalStepBarrelTracks',
+))
+
+jetCoreRegionalStepEndcap = jetCoreRegionalStep.clone(
+    src = 'jetCoreRegionalStepEndcapTracks',
+)
```

Comment:
It seems like we are converging on having the endcap part consistent with post-era `jetCoreRegionalStep` (while the DNN-based barrel part can stay more flexible/independent).
If so, then this `jetCoreRegionalStepEndcap` definition can be moved below after `fastSim.toModify(`